runtime/queries/kotlin/locals.scm

```diff
@@ -0,0 +1,83 @@
+;;; Imports
+
+(package_header
```

Comment:
This locals definition doesn't match helix/tree-sitter, we use different scopes: https://github.com/helix-editor/helix/blob/master/runtime/queries/rust/locals.scm

It's okay to just remove `locals.scm` and we can do them in a follow-up PR though

Reply:
Done